src/core/loop_induction_variables/include/noelle/core/InductionVariable.hpp

```diff
@@ -44,6 +44,25 @@ class InductionVariable {
                     LoopEnvironment &loopEnvironment,
                     ScalarEvolutionReferentialExpander &referentialExpander);
 
+  InductionVariable(LoopStructure *LS,
+                    InvariantManager &IVM,
+                    ScalarEvolution &SE,
+                    int stepMultiplier,
```

Comment:
Please avoid using "int", "long int", etc... and instead use "uint32_t", "int64_t", etc...